services/src/main/java/org/keycloak/authorization/protection/permission/PermissionTicketService.java

```diff
@@ -226,6 +226,52 @@ public Response find(@QueryParam("scopeId") String scopeId,
                 .build();
     }
 
+    @Path("/count")
+    @GET
+    @Produces("application/json")
+    public Response getResourceCount(@QueryParam("scopeId") String scopeId,
+                                       @QueryParam("resourceId") String resourceId,
+                                       @QueryParam("owner") String owner,
+                                       @QueryParam("requester") String requester,
+                                       @QueryParam("granted") Boolean granted,
+                                       @QueryParam("returnNames") Boolean returnNames) {
+        StoreFactory storeFactory = authorization.getStoreFactory();
+        PermissionTicketStore permissionTicketStore = storeFactory.getPermissionTicketStore();
+        Map<String, String> filters = new HashMap<>();
+
+        if (resourceId != null) {
```

Comment:
Line 238 - 267 have been copied from the find endpoint, so it's duplicate. Should this be separated into another method?

Reply:
Yeah, makes more sense.